setup.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+import { create } from 'hyper-sdk'
+import { parseArgs } from '@pkgjs/parseargs'
+
+const options = {
+  storage: { type: 'string', short: 's'}
+}
+
+const { storage } = parseArgs({ options }).values
+
+const sdk = await create({
+  storage: storage // TODO: Where should the default location for storage be?
+})
+
+console.log(sdk.publicKey.toString('hex'))
```

Comment:
What is the purpose of this utility?

Reply:
This utility was for getting the public key for the namespace without pushing or pulling.